Sources_v3/StreamChatUI/ChatChannelList/ChatChannelView.swift

```diff
@@ -0,0 +1,51 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+public protocol ChatChannelViewType: UIView {
+    var title: String? { get set }
+    var titleLabelTextColor: UIColor { get set }
+}
+
+open class ChatChannelView: UIView, ChatChannelViewType {
+    // MARK: - Subviews
+
+    private let titleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    // MARK: - API
+    
+    @objc public dynamic var titleLabelTextColor: UIColor {
```

Comment:
We have to use `@objc dynamic` to have dynamic dispatch happen. This allows to customize this value via appearance proxy for all instances of `ChatChannelView`:

```swift
ChatChannelView
 .appearance()
 .titleLabelTextColor = .red
``` 

However, we can still set a custom value for a particular view if it's needed:
```swift
let channelView = ChatChannelView(frame: .zero)
channelView.titleLabelTextColor = .black
```